client/src/app/pages/vulnerability-list/vulnerability-table.tsx

```diff
@@ -97,7 +95,7 @@ export const VulnerabilityTable: React.FC = () => {
                         rowIndex,
                       })}
                     >
-                      X
+                      {item.cwes.length}
```

Comment:
I think we should render not the length but the content of the array. If there are multiple elements we could join them using comma.